jinahub/encoders/text/DPRTextEncoder/README.md

```diff
@@ -0,0 +1,102 @@
+# ✨ DPRTextEncoder
+
+ **DPRTextEncoder** is a class that encodes text into embeddings using the DPR model from huggingface transformers.
+
+The **DPR** model was originally proposed in [Dense Passage Retrieval for Open-Domain Question Answering](https://arxiv.org/abs/2004.04906).
+
+This encoder supports both the DPR context and question encoders - you should specify which type you are using with the `encoder_type` parameter.
+
+The following parameters can be passed on initialization:
+
+- `pretrained_model_name_or_path`: Can be either:
+	- the model id of a pretrained model hosted inside a model repo
+		on huggingface.co.
+	- A path to a directory containing model weights, saved using
+		the transformers model's `save_pretrained()` method
+- `encoder_type`: Either `'context'` or `'question'`. Make sure this
+	matches the model that you are using.
+- `base_tokenizer_model`: Base tokenizer model. The possible values are
+	the same as for the ``pretrained_model_name_or_path`` parameters. If not
+	provided, the ``pretrained_model_name_or_path`` parameter value will be used
+- `title_tag_key`: The key under which the titles are saved in the documents'
+    tag property. It is recommended to set this property for context encoders,
+    to match the model pre-training. It has no effect for question encoders.
+- `max_length`: Max length argument for the tokenizer
+- `default_batch_size`: Default batch size for encoding, used if the
+	batch size is not passed as a parameter with the request.
+- `default_traversal_paths`: Default traversal paths for encoding, used if the
+	traversal path is not passed as a parameter with the request.
+- `device`: The device (cpu or gpu) that the model should be on.
+
+
+**Table of Contents**
+
+- [🌱 Prerequisites](#-prerequisites)
+- [🚀 Usages](#-usages)
+- [🎉️ Example](#%EF%B8%8F-example)
+- [🔍️ Reference](#%EF%B8%8F-reference)
+
+## 🌱 Prerequisites
+
+No prerequisites are required to run this executor.
```

Comment:
Aren't there requirements? Add a note that if you want to run the class directly, you need to install them

Reply:
Why would we do that? We don't have instructions on how to run locally anymore. I thought the idea is to steer users towards jinahub.

Reply:
that's true, but all the other executors say it, so we must've agree to have it written, no?

see https://github.com/jina-ai/executors/blob/d2dae5c1d008bd89ba5817b715b41fb7694af803/jinahub/encoders/image/BigTransferEncoder/README.md#L28

Reply:
We should rethink what goes in pre-requisites. I think instructions on how to download the models and mount volumes to containers would be more useful for most people. But that can be done in another PR